samples/bookinfo/src/details/details.rb

```diff
@@ -63,4 +74,29 @@ def get_book_details(id)
     }
 end
 
+def fetch_details_from_external_service(isbn, id)
+    uri = URI.parse('http://api.bookmooch.com')
```

Comment:
This is a bit unrealistic.. Since most people use HTTPS.

Reply:
This is just an example to demonstrate egress rules in a simplest way possible. I will write a blog about HTTP, and also about TCP/Mongo, and also about HTTPS. The first one would be about HTTP and would be the simplest one to understand and demonstrate. Note that in this case api.bookmooch.com provides HTTP API only, so this particular example is rather realistic.